src/items.js

```diff
@@ -0,0 +1,217 @@
+};
+
+const addComment = (event) => {
+  event.preventDefault();
+  const { parentElement } = event.target;
+  const nameInput = parentElement.querySelector('input');
+  const commentInput = parentElement.querySelector('textarea');
+  const name = nameInput.value;
+  const comment = commentInput.value;
+  const id = event.target.id.split('add-comment-')[1];
+
+  for (const i in items) { /* eslint-disable-line */
+    if (items[i].pokemon === id) {
+      const newComment = {
+        name,
+        date: formatDate(new Date()),
+        comment,
+      };
+      items[i].comments = items[i].comments.concat(newComment);
+
+      const commentsDiv = getElementById(`comments-${id}`);
+      const commentDiv = createDivWithClass('d-flex flex-column mb-1 py-1 border-bottom container');
+
+      const h4Element = createElementWithClass('h4', 'font-medium-1');
+      h4Element.textContent = newComment.name;
+
+      const h5Element = createElementWithClass('h5', 'font-small-2');
+      h5Element.textContent = newComment.date;
+
+      const pElement = createElementWithClass('p', 'font-small-3');
+      pElement.textContent = newComment.comment;
+
+      commentDiv.appendChild(h4Element);
+      commentDiv.appendChild(h5Element);
+      commentDiv.appendChild(pElement);
+
+      commentsDiv.appendChild(commentDiv);
+
+      const commentsCounter = getElementById(`comments-counter-${id}`);
+      commentsCounter.textContent = `Comments (${items[i].comments.length})`;
+
+      saveItems(items);
+
+      break;
+    }
+  }
+
+  nameInput.value = '';
+  commentInput.value = '';
+};
+
+const displayItems = async () => {
+  items = items || await loadItems();
+  const itemsContainerElement = getElementById('items-container');
+
+  const itemsCounterElement = getElementById('items-counter');
+  itemsCounterElement.textContent = `Pokémon (${items.length})`;
+
+  items.forEach((item) => {
+    const pkmn = item.pokemon;
+
+    const columnDiv = createDivWithClass('d-flex flex-column');
+    const idDiv = createDivWithId(pkmn);
+    const imageContDiv = createDivWithClass('image-cont');
+    const imgElement = createElement('img');
+    imgElement.setAttribute('src', item.image_url);
+    imgElement.setAttribute('alt', pkmn);
+
+    const h5Element = createElementWithClass('h5', 'mt-3');
+    h5Element.textContent = `${pkmn} `;
+
+    const spanElement = createElement('span');
+    const heartElement = createHeartElement(item);
+
+    spanElement.appendChild(heartElement);
+    h5Element.appendChild(spanElement);
+
+    const pElement = createElement('p');
+    pElement.id = `likes-${pkmn}`;
+    pElement.textContent = `${item.likes} likes`;
+
+    const commentModal = createDivWithClass('modal');
+    commentModal.innerHTML = `           
+      <div class="modal-content">
+        <span class="close">&times;</span>
+        <img src="${item.image_url}" class="w-25 container border border-dark mb-3"></img>
+        <h3 class="text-center mb-3">${pkmn}</h3>
```

Comment:
- [ ] As per project requirements, when the popup loads, the web app retrieves data from the selected API and shows details about the selected item. Kindly try to add a little more detail about the item in the comment pop-up. Check the example below.

your
![image](https://user-images.githubusercontent.com/53019916/128588634-aa59ea6f-7e8a-49e0-9962-2f0d203177c5.png)

Add details like this example
![image](https://user-images.githubusercontent.com/53019916/128588677-c7958627-7fe1-489d-9ca8-eab8703ef130.png)

Reply:
Thank you for the observation - we did miss this. We've updated it to retrieve more data about the Pokémon now:

![Screen Shot 2021-08-07 at 2 47 50 AM](https://user-images.githubusercontent.com/2223583/128591348-eb46bd18-f639-4dca-a836-aa0df14f4662.png)